localMatchApp/headers/homewindow.h

```diff
@@ -2,24 +2,76 @@
 #define HOMEWINDOW_H
 
 #include <QDialog>
+#include "ClassEvent.h"
 
 /*!
  * The class of user interface type is created to
  * be able to develop the main screen
  */
+
 namespace Ui {
 class HomeWindow;
 }
 
-class HomeWindow : public QDialog {
-  Q_OBJECT
+class HomeWindow : public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit HomeWindow(QWidget *i_parent = nullptr);
+    ~HomeWindow();
 
- public:
-  explicit HomeWindow ( QWidget *i_parent = nullptr );
-  ~HomeWindow ( );
+/*!
+ * The functions of button actions are declared so
+ * they can be used later
+ */
+private slots:
```

Comment:
Why private slots?

Reply:
Because they have to be of the slot type since the signals of the buttons need it to be able to give functionality to the button, if they are only private it does not work